docs/apriori_sbms.ipynb

```diff
@@ -0,0 +1,513 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(2)\n",
+    "out_index = ww_results.index.levels[0]\n",
+    "colors = sns.color_palette(\"dark\")\n",
+    "for i, key in enumerate(out_index):\n",
+    "    data = ww_results.loc[key]\n",
+    "    bic_data = data['bic'].to_numpy()\n",
+    "    norm_bic = bic_data - np.min(bic_data)\n",
+    "    like_data = data['likelihood'].to_numpy()\n",
+    "    norm_like = like_data - np.min(like_data)\n",
+    "    ax[0].scatter(data['bic'].index,\n",
```

Comment:
Can you plot these in green (with the appropriate seaborn color palette) with males as triangles and hermaphrodites as circles to be consistent with the graph stats plots?